<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RJINDAL" name="Test in FF" tests="5" failures="5" timestamp="9 Oct 2014 09:06:32 GMT" time="377.679" errors="0">
  <testcase name="DuplicateClubRegistrationUsingCSVFile" time="68.907" classname="com.seleniumtests.test.ClubReg">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: expected:<false> but was:<true>
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.seleniumtests.core.CustomVerification.checkForVerificationErrors(CustomVerification.java:102)
	at com.seleniumtests.test.ClubReg.DuplicateClubRegistrationUsingCSVFile(ClubReg.java:113)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase>
  <testcase name="DuplicateClubRegistrationUsingCSVFile" time="65.598" classname="com.seleniumtests.test.ClubReg">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;java.lang.AssertionError: expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: expected:<true> but was:<false>java.lang.AssertionError: expected:<false> but was:<true>
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.seleniumtests.core.CustomVerification.checkForVerificationErrors(CustomVerification.java:102)
	at com.seleniumtests.test.ClubReg.DuplicateClubRegistrationUsingCSVFile(ClubReg.java:113)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase>
  <testcase name="DuplicateClubRegistrationUsingCSVFile" time="66.046" classname="com.seleniumtests.test.ClubReg">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;java.lang.AssertionError: expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: expected:<true> but was:<false>java.lang.AssertionError: expected:<false> but was:<true>
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.seleniumtests.core.CustomVerification.checkForVerificationErrors(CustomVerification.java:102)
	at com.seleniumtests.test.ClubReg.DuplicateClubRegistrationUsingCSVFile(ClubReg.java:113)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase>
  <testcase name="DuplicateClubRegistrationUsingCSVFile" time="60.179" classname="com.seleniumtests.test.ClubReg">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: expected:<false> but was:<true>
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.seleniumtests.core.CustomVerification.checkForVerificationErrors(CustomVerification.java:102)
	at com.seleniumtests.test.ClubReg.DuplicateClubRegistrationUsingCSVFile(ClubReg.java:113)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase>
  <testcase name="DuplicateClubRegistrationUsingCSVFile" time="71.137" classname="com.seleniumtests.test.ClubReg">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: expected:<false> but was:<true>
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.seleniumtests.core.CustomVerification.checkForVerificationErrors(CustomVerification.java:102)
	at com.seleniumtests.test.ClubReg.DuplicateClubRegistrationUsingCSVFile(ClubReg.java:113)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase>
</testsuite>
